scipy/sparse/tests/test_base.py

```diff
@@ -756,23 +756,39 @@ def test_sum(self):
         matrices = [dat_1, dat_2, dat_3, dat_4, dat_5]
 
         def check(dtype, j):
+            if j == 1:
+                # At the lowest tested precision for floats and complex
+                # numbers, the sparse matrix sum method can lead to different
+                # results in the lower decimals from the dense matrix sum
+                # method. This necessitaties a more lenient comparison between
+                # the resulting matrices.
+                if dtype == np.complex64:
+                    decimal = 4
+                else:
+                    decimal = 5
```

Comment:
This [Travis test failure](https://travis-ci.org/scipy/scipy/jobs/112727039) indicates that these need to be loosened even more.

Alternatively, we could compute the full dense sums as `dat[dat!=0].sum()` and sidestep most of the precision issues.